openmetrics/src/main/java/io/airlift/openmetrics/MetricsConfig.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.openmetrics;
+
+import com.google.common.collect.ImmutableList;
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+
+import java.util.List;
+
+public class MetricsConfig
+{
+    private List<String> jmxDomains = ImmutableList.of();
+
+    public List<String> getJmxDomains()
+    {
+        return jmxDomains;
+    }
+
+    @Config("openmetrics.jmx-domains")
```

Comment:
It'd be more powerful and generic for these to be objectname patterns. It would allow for more complex patterns that filter out certain object names that are not interesting, or to name object names explicitly.